common/scripted_effects/wc_scripted_effects.txt

```diff
@@ -377,7 +377,7 @@ add_class_trait_effect = {
 		}
 		if = {
 			limit = {
-				true_religion = kaldorei_religion
+				religion = kaldorei_religion
```

Comment:
Conflicted lines. I moved them to wc_class_scripted_effects.txt. I'll implement your changes there myself.

Reply:
Done